temba/urls.py

```diff
@@ -50,7 +50,7 @@
 
 # import any additional urls
 for app in settings.APP_URLS:  # pragma: needs cover
-    importlib.import_module(app)
+    urlpatterns.append(url(r"^", include(app)))
```

Comment:
This didn't seem like it was registering the urls... seems like it's just importing the module ?

Reply:
Should we be configuring our `APP_URLS` as below?
```python
APP_URLS = ['mycustomapp.urls', ]
```
 An import of the URLs module should register URLs, you do not need to explicitly do `urlpatterns.append`. Did you test this and it was not working?

Reply:
Yes, I tested this and it wasn't picking up the urls. Might it be an issue with how the urls are declared in `ona-oidc` ?